plugins/Diagnostics/Diagnostic/MemoryLimitCheck.php

```diff
@@ -41,7 +41,10 @@ public function execute()
         $memoryLimit = SettingsServer::getMemoryLimitValue();
         $comment = $memoryLimit . 'M';
 
-        if ($memoryLimit >= $this->minimumMemoryLimit) {
+        if(false === $memoryLimit) {
+            $status = DiagnosticResult::STATUS_OK;
+            $comment = $this->translator->translate('Installation_SystemCheckNoMemoryLimitSet');
```

Comment:
You also need to add the default text for this to `plugins\Installation\lang\en.json`

Reply:
Ok done.